specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -2,199 +2,242 @@
-* The client sends the channel URL to the app as part of the channel request in step 4)
-* The app sends the channel URL to it’s app server and saves it.
-
-The next step involves the application actually Pushing down a payload from the App Service to the device:
-* The App Server requests an access token from MSA using the AppSID and client secret retrieved from the developer registration process with dev center.
-* MSA returns a token to the app server
-* App server posts a notification’s to WNS’s NW with the channel url retrieved from WPN client and the access token in step 2)
-* NW verifies the app token against AppSID extracted from the channel URL
-* NW sends the Post request to WNS’s NN which contains the persistent socket with the client
-* WNS’s NN maps the channel URL to the correct deviceId by decoding the deviceId from the channel URL and sends an NFY to the correct client.
-* WPN client receives the NFY and processes the channelId in the payload to retrieve the correct AppId from the WPN database.
-* WPN client routes the notification to the appropriate app. If it needs to interface with the App, it uses BI, callbacks or WNF depending on the App Registration. If the payload is a toast, it gets directly routed to the Shell.
+* [The Windows Push Notification Services (WNS) Overview](https://docs.microsoft.com/windows/uwp/design/shell/tiles-and-notifications/windows-push-notification-services--wns--overview)
+
+![Legacy Flow For Reunion](Legacy.png)
 
 ## The problems today
 
-**Portal Registration**: The entire Registration flow for Push is dependent on registering an app in the partner centre portal by [associating an app with the Windows Store](https://docs.microsoft.com/en-us/azure/notification-hubs/notification-hubs-windows-store-dotnet-get-started-wns-push-notification). As a result, non-store provisioned apps like Win32 cannot avail of this feature even if they want to.
+**Portal Registration**: The entire Registration flow for Push is dependent on registering an app in the <br/>
+partner center portal by [associating an app with the Windows Store](https://docs.microsoft.com/azure/notification-hubs/notification-hubs-windows-store-dotnet-get-started-wns-push-notification). As a result, non-store provisioned <br/>
+apps like Win32 cannot avail of this feature even if they want to.
 
-**Client Registration**: Unpackaged 3rd party applications do not have a way today to register with the client because the API is private. The only way to get an implicit client side registration is to package the app as an MSIX or SSP which may not satisfy 3rd party requirements.
+**Client Registration**: Unpackaged 3rd party applications do not have a way today to register with the <br/>
+client because the API is private. The only way to get an implicit client side registration is to package <br/>
```

Comment:
don't advertise the private API, simply say "Unpackaged applications are not supported in this design."